recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
   run:
     - python >=3.8,<4.0
     - shapely >=1.8
-    - matplotlib-base >=3.5
+    - matplotlib-base >=3.6,<3.7
```

Comment:
@betolink I've made this change since it's in the pyproject.toml upstream, but it looks like the binder env is much less restrictive. we should maybe circle back and consider unrestricting some of these.